### PR TITLE
branch to merge into master with custom keybindings

### PR DESCRIPTION
**Pull request by ewaugh2**

### CODE REVIEW

File: ECS189L/Assets/Scripts/Input/InputManager.cs [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        buttonKeys["Right4"] = KeyCode.H;
+
+        buttonKeys["Attack1"] = KeyCode.Space;
+        buttonKeys["Attack2"] = KeyCode.Q;
+        buttonKeys["Attack3"] = KeyCode.U;
+        buttonKeys["Attack4"] = KeyCode.R;
+    }
+
+    
+    void Start()
```
Comment (FernandoPieressa) [inline ECS189L/Assets/Scripts/Input/InputManager.cs]: Delete if not used

File: ECS189L/Assets/Scripts/Input/InputManager.cs [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    {
+        
+    }
+
+    // Update is called once per frame
+    void Update()
```
Comment (FernandoPieressa) [inline ECS189L/Assets/Scripts/Input/InputManager.cs]: Delete if not used

File: ECS189L/Assets/Scripts/Input/SetKeyMainMenu.cs [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+    public void Update()
+    {
+        
+    }
+
+    void OnGUI()
+    {
+        keyEvent = Event.current;
+        
+        if(keyEvent.isKey && waitingForKey)
```
Comment (FernandoPieressa) [inline ECS189L/Assets/Scripts/Input/SetKeyMainMenu.cs]: be constant with spaces

File: ECS189L/Assets/Scripts/Input/InputManager.cs [hunk 4]
```diff
@@ -0,0 +1,74 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class InputManager : MonoBehaviour
+{
+    public static InputManager IM;
+
+    [SerializeField] public Dictionary<string, KeyCode> buttonKeys;
```
Comment (FernandoPieressa) [inline ECS189L/Assets/Scripts/Input/InputManager.cs]: Serialized fields should be private

**Review by FernandoPieressa — CHANGES_REQUESTED** [review]

File: ECS189L/Assets/Scripts/Input/InputManager.cs (hunk 1 above)
Comment (ewaugh2) [reply ECS189L/Assets/Scripts/Input/InputManager.cs]: Done

File: ECS189L/Assets/Scripts/Input/InputManager.cs (hunk 2 above)
Comment (ewaugh2) [reply ECS189L/Assets/Scripts/Input/InputManager.cs]: Done

File: ECS189L/Assets/Scripts/Input/SetKeyMainMenu.cs (hunk 3 above)
Comment (ewaugh2) [reply ECS189L/Assets/Scripts/Input/SetKeyMainMenu.cs]: Done

File: ECS189L/Assets/Scripts/Input/InputManager.cs (hunk 4 above)
Comment (ewaugh2) [reply ECS189L/Assets/Scripts/Input/InputManager.cs]: Fixed

**Pull request merged**